### DISCUSSION TITLE
[Core] Coordination

### DISCUSSION THREAD

**Discussion by atomiks** [floating-ui/floating-ui#2719] [General]
When multiple floating elements (controlled by Floating UI) want to avoid each other, the [collidables technique](https://github.com/floating-ui/floating-ui/issues/1440) means floating elements positioned first take precedence, and so only the first one is positioned in a nice layout. Later ones are pushed aside or positioned in a less than ideal way.

Coordination would solve this by giving the positioning the context of a group, meaning the group as a whole is positioned in a flexbox-like manner so each floating element is nicely laid out.

**Comment by AvWoN** [comment]
Hello Atomiks, is there any movement on this since April?

Cheers,
David

**Comment by BitBrit** [comment]
This is exactly what I am looking for; a coordinated group of floating elements positioned around 1 or more targets.  I hope this lands soon.

**Reply by atomiks** [reply]
It's not a trivial problem unfortunately. I haven't got too much time to work on this, so it'll need to come from external contributors.
